openedx/core/djangoapps/profile_images/images.py

```diff
@@ -127,51 +120,102 @@ def validate_uploaded_image(uploaded_file):
     uploaded_file.seek(0)
 
 
-def _get_scaled_image_file(image_obj, size):
+def _crop_image_to_square(image):
+    """
+    Given a PIL.Image object, return a copy cropped to a square around the
+    center point with each side set to the size of the smaller dimension.
+    """
+    width, height = image.size
+    if width != height:
+        side = width if width < height else height
+        left = (width - side) // 2
+        top = (height - side) // 2
+        right = (width + side) // 2
+        bottom = (height + side) // 2
+        image = image.crop((left, top, right, bottom))
+    return image
+
+
+def _set_color_mode_to_rgb(image):
+    """
+    Given a PIL.Image object, return a copy with the color mode set to RGB.
+    """
+    return image.convert('RGB')
+
+
+def _scale_image(image, size):
+    """
+    Given a PIL.Image object, get a resized copy with each side being `size`
+    pixels long.  The scaled image will always be square.
+    """
+    return image.resize((size, size), Image.ANTIALIAS)
```

Comment:
It's a bit confusing to see `size` as a `width, height` tuple above (https://github.com/edx/edx-platform/blob/cdyer/exif-rotation/openedx/core/djangoapps/profile_images/images.py#L128), but an integer side length here. Could this value be renamed `side_length` or something similar?